src/test/java/com/afornalik/ox/controller/TestSkirmishController.java

```diff
@@ -0,0 +1,69 @@
+package com.afornalik.ox.controller;
+
+import com.afornalik.ox.model.board.Board;
+import com.afornalik.ox.model.board.FieldStatus;
+import com.afornalik.ox.model.board.OutOfBoardException;
+import com.afornalik.ox.model.player.PlayerContainer;
+import com.afornalik.ox.view.UIConsole;
+import com.afornalik.ox.view.UIOperations;
+import org.mockito.Mockito;
+import org.testng.annotations.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Test
+public class TestSkirmishController {
```

Comment:
I am not convinced to the names you use. I could not guess what test is checking just by reading it's name. I guess it is probably because you are only telling what should happen but not when.

Reply:
Changed to TestMatchController.